lib/Web/Server.hs

```diff
@@ -44,12 +44,13 @@ import Servant.Server
 data CofferServerError = CofferServerError
   { cseError :: Text
   , cseCode :: Int
+  , cseResult :: Text
```

Comment:
Hmm, I don't understand the purpose of this field. Could you please explain it's function?

Reply:
Check my answer above

Reply:
Removed anyway